{{ cookiecutter.project_shortname }}/.travis.yml

```diff
@@ -24,12 +24,12 @@ python:
 before_install:
   - "travis_retry pip install --upgrade pip setuptools py"
   - "travis_retry pip install twine wheel coveralls requirements-builder"
-  - "requirements-builder --level=min setup.py > .travis-lowest-requirements.txt"
-  - "requirements-builder --level=pypi setup.py > .travis-release-requirements.txt"
-  - "requirements-builder --level=dev --req requirements-devel.txt setup.py > .travis-devel-requirements.txt"
+  - "requirements-builder --level=min setup.py > built-requirements-lowest.txt"
```

Comment:
If we rename the options in requirements-builder then this should be done in a backward compatible way, that will allow the old values to still be used and issuing a pending deprecation warning. Otherwise we have a lot of ``.travis.yml`` files to fix very fast. I'm probably just stating the obvious here :-)